src/Libraries/Microsoft.Extensions.AI.Abstractions/SpeechToText/SpeechToTextResponseUpdateExtensions.cs

```diff
@@ -0,0 +1,153 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Shared.Diagnostics;
+
+namespace Microsoft.Extensions.AI;
+
+/// <summary>
+/// Provides extension methods for working with <see cref="SpeechToTextResponseUpdate"/> instances.
+/// </summary>
+[Experimental("MEAI001")]
+public static class SpeechToTextResponseUpdateExtensions
+{
+    /// <summary>Combines <see cref="SpeechToTextResponseUpdate"/> instances into a single <see cref="SpeechToTextResponse"/>.</summary>
+    /// <param name="updates">The updates to be combined.</param>
+    /// <param name="coalesceContent">
+    /// <see langword="true"/> to attempt to coalesce contiguous <see cref="AIContent"/> items, where applicable,
+    /// into a single <see cref="AIContent"/>. When <see langword="false"/>, the original content items are used.
+    /// The default is <see langword="true"/>.
+    /// </param>
+    /// <returns>The combined <see cref="SpeechToTextResponse"/>.</returns>
+    public static SpeechToTextResponse ToSpeechToTextResponse(
+        this IEnumerable<SpeechToTextResponseUpdate> updates, bool coalesceContent = true)
```

Comment:
Remove the bool parameter as was done with ToChatResponse. The coalescing always happens now.